src/wp-content/themes/twentysixteen/inc/back-compat.php

```diff
@@ -4,7 +4,7 @@
  *
  * Prevents Twenty Sixteen from running on WordPress versions prior to 4.4,
```

Comment:
I think at the end of this line we need an additional `WP-`?

Reply:
I didn't since it already says WordPress here, but I'm fine with being consistent too.

Reply:
Actually looking at the whole PR I think I did?

Reply:
> Actually looking at the whole PR I think I did?

Not sure I understand?

Reply:
With this sentence you're actually right, "WordPress" make it clear already